joplin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+{% if page_types_restricted %}
+    {% if page_type_string == "base.departmentpage" %}
+    {% trans "Add a department that offers this page" as choose_str %}
+    {% include "wagtailadmin/shared/header.html" with title=choose_str  search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+    {% elif page_type_string == "base.informationpage,base.servicepage" %}
+        {% trans "Select a page for the guide" as choose_str %}
+        {% include "wagtailadmin/shared/header.html" with title=choose_str  search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+
+    {% else %}
+        {% trans "Choose" as choose_str %}
```

Comment:
I know this is a fallback and we don't have an example where this happens, but having simply "choose" nested deeper in the conditional than "choose a page" threw me off at first

Reply:
So this one renders as "Choose" {{ page_type_string}} which can be a list of types. It's generic but more specific than just Choose